topics/epigenetics/metadata.yaml

```diff
@@ -3,6 +3,7 @@ type: "use"
 title: "DNA Methylation data analysis"
 summary: "DNA methylation is an epigenetic mechanism used by higher eukaryotes and involved in eg. gene expression, X-Chromosome inactivating, imprinting, and gene silencing of germ-line specific gene and repetitive elements."
 docker_image: ""
+edam_ontology: "topic_3173"
```

Comment:
`topic_` is part of the id, there are also `format_N`, `data_N` and `operation_N` ids.